userguide/plugins.rst

```diff
@@ -623,9 +623,10 @@ Official Plugins
 ----------------
 
 :numref:`table %s <plugins-official-plugins>` lists and describes all
-plugins supported by iXsystems. Adding "unofficial" plugins to %brand%
-is supported by following the process outlined in
-:ref:`Create a Plugin <Creating Plugins>`.
+plugins supported by iXsystems. Access the official plugins by choosing
+*iXsystems* from the :guilabel:`Browse a Collection` drop-down. Adding
+"unofficial" plugins to %brand% is supported by following the process
```

Comment:
This sentence is long, complex, and unclear.  Why is unofficial quoted?  "supported by following the process outlined" is complicated, and can be cleared up by rearranging and removing unneeded words: "Unofficial plugins can be added by..." Also, which process in that section?